apps/validate/validate.py

```diff
@@ -128,6 +128,19 @@ def _validate_company_codes(self, *args):
         """Ignore company codes."""
         pass
 
+    def _validate_media_metadata(self, validate, associations_field, associations):
```

Comment:
Current implementation checks for max_length as well (see here https://github.com/superdesk/superdesk-aap/blob/master/client/superdesk.config.js#L6) and in the client-core config only required field validation was there. 
@jerome-poisson Can you please confirm that this feature is not removed?

Reply:
@mdhaman nothing has been removed, but `validatorMediaMetadata` must be moved from client to backend in `superdesk-aap`. The client will then get the whole object from backend.

Also only `required` is checked in backend so far, I'll add maxlenght if it's present (but it's still checked in client if it was before).